src/WaitingList.js

```diff
@@ -0,0 +1,49 @@
+
+
+function WaitingList({location, initialState, onClickTodoItem, onDeleteTodoItem}) {
+
+    this.state = {...initialState};
+
+    this.setState = (nextState) => {
+        this.state = {
+            ...this.state,
+            ...nextState
+        }
+
+        this.render();
+    }
+
+    this.render = (waitingTodo) => {
```

Comment:
```suggestion
    this.render = (waitingTodos) => {
```

복수형으로 지으면 좋을 것 같습니다!